test/upload-download.test.js

```diff
@@ -338,15 +347,16 @@ describe('storage service', function() {
       .set('Accept', 'application/json')
       .expect('Content-Type', /json/)
       .expect(200, function(err, res) {
-        done();
+        done(err);
```

Comment:
>`if (err)`
I don't think if thats needed, if there is no `err`, `err` will be undefined in the following line:
```
done(err);
```